ui/src/components/Description/Description.css

```diff
@@ -54,6 +54,13 @@ h3 {
 .hub-readme {
   font-family: Helvetica;
 }
+.hub-readme > table,
+th,
+td {
+  border: 1px solid black;
+  text-align: center !important;
+  padding: 0em 0.3em 0em 0.3em;
```

Comment:
can we increase the padding more? maybe `1em` each. Right now the table is looking like this :arrow_down: 
![image](https://user-images.githubusercontent.com/26500025/105384583-678ab680-5c38-11eb-8603-e5d7d658ac1c.png)

and with `1em` each :arrow_down: 
![image](https://user-images.githubusercontent.com/26500025/105384756-a3258080-5c38-11eb-9b36-3109789124c4.png)